pyxem/utils/calibration.py

```diff
@@ -353,7 +362,11 @@ def _get_radial_range(self, radial_range=None):
             radial_range = (0, max_range)
         return radial_range
 
-    def get_slices1d(self, npt, radial_range=None):
+    def get_slices1d(
+        self,
+        npt: int,
+        radial_range: typing.Union[typing.Sequence[float], typing.Sequence[int]] = None,
```

Comment:
Explicitly supporting both float and int is done a lot of places, I see. Maybe we could define e.g. `Number = int | float` in a little utils.typing module for example.

Reply:
That's a good idea :)